feature_engine/variable_manipulation.py

```diff
@@ -64,4 +139,5 @@ def _find_all_variables(X, variables=None):
         if len(set(variables).difference(X.columns)) != 0:
             raise TypeError("Some variables are not present in the dataset. Please check your variable"
```

Comment:
should we use a genexp?

Reply:
Using genexp, the test does not fail. So, no problem.